core/os/android/adb/commands.go

```diff
@@ -331,9 +332,15 @@ func (b *binding) QueryPerfettoServiceState(ctx context.Context) (*device.Perfet
 		result.GpuProfiling = gpu
 	}
 
-	if b.Instance().GetConfiguration().GetOS().GetName() == "R" {
-		// TODO(b/146384733): Change this to API version when it releases
-		gpu.HasFrameLifecycle = true
+	if sdk,err := b.SystemProperty(ctx, "ro.build.version.sdk"); err == nil {
```

Comment:
I think we can wait until `GetAPI()` returns the api level for Android 11

Reply:
Charlie is partly blocked on this, so I would prefer this change sooner rather than later.  Also, this change should be able avoid something similar happening again next year.